kasa/cli.py

```diff
@@ -102,7 +102,10 @@ def __call__(self, *args, **kwargs):
             asyncio.get_event_loop().run_until_complete(self.main(*args, **kwargs))
         except Exception as ex:
             echo(f"Got error: {ex!r}")
-            raise
+            if "--debug" in sys.argv or "-d" in sys.argv:
+                raise
+            else:
+                echo("Run with --debug enabled to see stacktrace")
```

Comment:
```suggestion
            echo("Run with --debug enabled to see stacktrace")
```
No need for else clause due to raise

Reply:
Fixed